src/components/LessonLink/LessonLink.scss

```diff
@@ -0,0 +1,55 @@
+@use "../../styles/variables.scss" as *;
+@use "../../styles/typography.scss" as *;
+
+.container {
+  width: 46.938rem;
+  height: 4.938rem;
+  display: flex;
+  justify-content: space-between;
+
+  &__course-info {
+    width: fit-content;
+    height: fit-content;
+    display: flex;
+    flex-direction: column;
+  }
+
+  &__title {
+    display: flex;
+    justify-content: space-between;
+    color: $color-teal;
+    margin-bottom: 0.438rem;
+    gap: 1.323rem;
+  }
+
+  &__title-header {
+    color: $color-grey-normal;
+  }
+
+  &__time {
+    display: flex;
+    color: $color-grey-normal;
+    margin-left: 2.5rem;
+    gap: 0.469rem;
+  }
+
+  &__time-header {
+    color: $color-grey-normal;
+  }
+
+  &__arrow {
+    height: 3.125rem;
+    width: 3.125rem;
+    border-radius: 50%;
+    background-color: rgba($color-blue-navy-dark, 0.1);
+    color: rgba($color-blue-navy-dark, 0.1);
+    display: flex;
+    margin-top: 0.063px;
```

Comment:
Can you please made sure any pxs are converted to REM please

Reply:
converted px to rem